src/main/java/org/sonarlint/intellij/ui/tree/FindingTreeSummary.kt

```diff
@@ -74,6 +88,13 @@ class FindingTreeSummary(private val project: Project, private val treeContentKi
                 sinceText
             )
         }
+        
+        return if (scopeSuffix.isNotEmpty() && filesCount > 1) {
+            // Replace "in X files" with scope-specific text when we have a scope suffix
+            baseText.replace(Regex("in \\d+ files?"), scopeSuffix)
```

Comment:
Isn't there a way to set the text appropriately from the start instead of replacing it?